sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils fdo-mime multilib
+
+MY_P_AMD64="${P}-linux-x86_64"
+MY_P_X86="${P}-linux-i486"
+
+DESCRIPTION="A free research management tool for desktop and web"
+HOMEPAGE="http://www.mendeley.com/"
+SRC_URI="amd64? ( ${MY_P_AMD64}.tar.bz2 )
+	x86? ( ${MY_P_X86}.tar.bz2 )
+	amd64-linux? ( ${MY_P_AMD64}.tar.bz2 )
+	x86-linux? ( ${MY_P_X86}.tar.bz2 )"
+
+LICENSE="Mendeley-terms"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RESTRICT="fetch"
+
+DEPEND=""
+RDEPEND="
+	>=dev-qt/qtcore-4.6:4
```

Comment:
`dev-qt/qtcore-4.6` is not even in gx86 anymore, just replace it by `dev-qt/qtcore:4`.